inst/codec_data/parcel/README.md

```diff
@@ -0,0 +1,5 @@
+# Parcel Characteristics
+
+Census tract-level measures of parcel characteristics for all residential parcles in Hamilton County, Ohio. Tract-level measures are derived from the data packages stored in the [`parcel` repository](https://github.com/geomarker-io/parcel): `cagis_parcels`, `auditor_online_parcels`, and `property_code_enforcements`. View the metadata for each of these data packages for more information about their sources. 
+
```

Comment:
Add what versions of each package this uses?

Version 0.1.0 of the `parcel` CoDEC data resource harmonizes `cagis_parcels-v1.1.0`, `auditor_online_parcels-v0.2.0`, and `property_code_enforcements-v1.0.0`.